### PR TITLE
[ZEVMIDST-206] Update sign in view

### PR DESCRIPTION
**Pull request by kacpersmal**
https://informedsolutions.atlassian.net/browse/ZEVMITSD-206

### CODE REVIEW

File: src/common/DfT.ZEV.Common.MVC.Authentication/Areas/Authentication/Views/Account/SignIn.cshtml [hunk 1]
```diff
@@ -10,61 +10,65 @@
...
+            body="Your email has been changed. Enter your new email address and password to sign in.">
+        </vc:notification-banner>
+    }
+    
+    
+    <h1 class="govuk-heading-l">@pageTitle</h1>
+    
+    @using (Html.BeginForm())
+    {   
+        <govuk-input asp-for="@Model.Email" inputmode="email" autocomplete="true" data-test="signin-email">
```
Comment (dae-ne) [inline src/common/DfT.ZEV.Common.MVC.Authentication/Areas/Authentication/Views/Account/SignIn.cshtml:53]: check if the `autocomplete="true"` value is correct

File: src/common/DfT.ZEV.Common.MVC.Authentication/Areas/Authentication/Views/Account/SignIn.cshtml [hunk 2]
```diff
@@ -10,61 +10,65 @@
...
+            <govuk-input-label>Email address</govuk-input-label>
+        </govuk-input>
+    
+        <govuk-input asp-for="@Model.Password" type="password" autocomplete="true" data-test="signin-password">
```
Comment (dae-ne) [inline src/common/DfT.ZEV.Common.MVC.Authentication/Areas/Authentication/Views/Account/SignIn.cshtml:57]: check if the `autocomplete="true"` value is correct

File: src/common/DfT.ZEV.Common.MVC.Authentication/Areas/Authentication/Views/Account/SignIn.cshtml (hunk 1 above)
Comment (kacpersmal) [reply src/common/DfT.ZEV.Common.MVC.Authentication/Areas/Authentication/Views/Account/SignIn.cshtml]: updated

File: src/common/DfT.ZEV.Common.MVC.Authentication/Areas/Authentication/Views/Account/SignIn.cshtml (hunk 2 above)
Comment (kacpersmal) [reply src/common/DfT.ZEV.Common.MVC.Authentication/Areas/Authentication/Views/Account/SignIn.cshtml]: updated

**Review by dae-ne — APPROVED** [review]

**Pull request merged**